entry.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+module.exports = Entry;
```

Comment:
Since we are making a breaking change. It would be nice to futureproof this.

I added a `LogMessage` data structure to `debug-logtron` ( https://github.com/Raynos/debug-logtron/blob/master/log-message.js#L17-L31 ).

It might be nice to have the same fields as that data structure, with the new extra `path` field.